### PR TITLE
updated version and re-added run parameter

### PR DESCRIPTION
**Pull request by pbek**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243815352%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243816658%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243817054%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243819290%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243860953%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243976950%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23discussion_r77128246%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23discussion_r77146393%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-244068787%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-244125552%22%2C%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-244317043%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23issuecomment-243815352%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22For%20some%20reason%20it%20fails%20to%20build%20-%20is%20it%20a%20missing%20build%20package%3F%20https%3A//travis-ci.org/ubuntu/snappy-playpen/builds/156561605%22%2C%20%22created_at%22%3A%20%222016-08-31T16%3A10%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1346979%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dholbach%22%7D%7D%2C%20%7B%22body%22%3A%20%22That%27s%20strange...%20%60xmlpatterns%60...%20It%20just%20worked%20locally%20and%20on%20Launchpad%20%3Chttps%3A//code.launchpad.net/%7Epbek/%2Bsnap/qownnotes%3E.%22%2C%20%22created_at%22%3A%20%222016-08-31T16%3A14%3A57Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22%60libqt5xmlpatterns5%60%20and%20%60libqt5xmlpatterns5-dev%60%20are%20even%20included%20as%20packages...%22%2C%20%22created_at%22%3A%20%222016-08-31T16%3A16%3A13Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22Sorry%2C%20my%20bad.%20There%20needed%20to%20be%20made%20other%20changes%20too...%22%2C%20%22created_at%22%3A%20%222016-08-31T16%3A23%3A20Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20don%27t%20understand%20why%20it%20is%20failing%2C%20it%20is%20the%20current%20%3Chttps%3A//git.launchpad.net/%7Epbek/qownnotes-snap/tree/snapcraft.yaml%3E%22%2C%20%22created_at%22%3A%20%222016-08-31T18%3A43%3A27Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22where%20does%20that%20error%20come%20from%3A%20%60%27ascii%27%20codec%20can%27t%20encode%20character%20%27%5C%5Cu29f8%27%20in%20position%2024%3A%20ordinal%20not%20in%20range%28128%29%60%3F%22%2C%20%22created_at%22%3A%20%222016-09-01T05%3A04%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22Does%20everything%20work%20now%3F%22%2C%20%22created_at%22%3A%20%222016-09-01T12%3A51%3A36Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1346979%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dholbach%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40dholbach%2C%20the%20local%20build%20and%20the%20build%20on%20Launchpad%20succeeded.%20Thanks%20a%20lot%21%22%2C%20%22created_at%22%3A%20%222016-09-01T15%3A57%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%2C%20%7B%22body%22%3A%20%22Nice%20work%21%22%2C%20%22created_at%22%3A%20%222016-09-02T08%3A38%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1346979%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dholbach%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20c5184564c8c9304e12233058e279b8979fe23ca4%20qownnotes/snapcraft.yaml%2053%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ubuntu/snappy-playpen/pull/231%23discussion_r77128246%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Can%20you%20change%20this%20to%20%60%5Bdesktop-qt5%5D%60%3F%22%2C%20%22created_at%22%3A%20%222016-09-01T07%3A50%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1346979%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dholbach%22%7D%7D%2C%20%7B%22body%22%3A%20%22Interesting%2C%20that%20worked%21%20I%20will%20try%20to%20build%20with%20that%20locally%20and%20on%20Launchpad%20too%21%20Thanks%20a%20lot%2C%20%40dholbach%21%22%2C%20%22created_at%22%3A%20%222016-09-01T10%3A01%3A15Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/1798101%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/pbek%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20qownnotes/snapcraft.yaml%3AL38-47%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/231#issuecomment-243815352'>General Comment</a></b>
- <a href='https://github.com/dholbach'></a> For some reason it fails to build - is it a missing build package? https://travis-ci.org/ubuntu/snappy-playpen/builds/156561605
- <a href='https://github.com/pbek'></a> That's strange... `xmlpatterns`... It just worked locally and on Launchpad <https://code.launchpad.net/~pbek/+snap/qownnotes>.
- <a href='https://github.com/pbek'></a> `libqt5xmlpatterns5` and `libqt5xmlpatterns5-dev` are even included as packages...
- <a href='https://github.com/pbek'></a> Sorry, my bad. There needed to be made other changes too...
- <a href='https://github.com/pbek'></a> I don't understand why it is failing, it is the current <https://git.launchpad.net/~pbek/qownnotes-snap/tree/snapcraft.yaml>
- <a href='https://github.com/pbek'></a> where does that error come from: `'ascii' codec can't encode character '\u29f8' in position 24: ordinal not in range(128)`?
- <a href='https://github.com/dholbach'></a> Does everything work now?
- <a href='https://github.com/pbek'></a> @dholbach, the local build and the build on Launchpad succeeded. Thanks a lot!
- <a href='https://github.com/dholbach'></a> Nice work!
- [ ] <a href='#crh-comment-Pull c5184564c8c9304e12233058e279b8979fe23ca4 qownnotes/snapcraft.yaml 53'></a> &nbsp;<b><a href='https://github.com/ubuntu/snappy-playpen/pull/231#discussion_r77128246'>File: qownnotes/snapcraft.yaml:L38-47</a></b>
- <a href='https://github.com/dholbach'></a> Can you change this to `[desktop-qt5]`?
- <a href='https://github.com/pbek'></a> Interesting, that worked! I will try to build with that locally and on Launchpad too! Thanks a lot, @dholbach!

<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/231?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ubuntu/snappy-playpen/pull/231?approve=1'></a>&nbsp;<a href='https://github.com/ubuntu/snappy-playpen/pull/231'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by didrocks** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ubuntu/snappy-playpen/pull/231'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dholbach** [comment]
For some reason it fails to build - is it a missing build package? https://travis-ci.org/ubuntu/snappy-playpen/builds/156561605

**Comment by pbek** [comment]
That's strange... `xmlpatterns`... It just worked locally and on Launchpad <https://code.launchpad.net/~pbek/+snap/qownnotes>.

**Comment by pbek** [comment]
`libqt5xmlpatterns5` and `libqt5xmlpatterns5-dev` are even included as packages...

**Comment by pbek** [comment]
Sorry, my bad. There needed to be made other changes too...

**Comment by pbek** [comment]
I don't understand why it is failing, it is the current <https://git.launchpad.net/~pbek/qownnotes-snap/tree/snapcraft.yaml>

**Comment by pbek** [comment]
where does that error come from: `'ascii' codec can't encode character '\u29f8' in position 24: ordinal not in range(128)`?

File: qownnotes/snapcraft.yaml [hunk 1]
```diff
@@ -40,6 +38,9 @@ parts:
       - libqt5svg5
       - libqt5network5
       - libqt5xml5
+      - libqt5xmlpatterns5
       - libqt5printsupport5
       - libqt5declarative5
-    after: [qt5conf]
+    stage:
+      - -usr/share/pkgconfig
+    after: [desktop/qt5]
```
Comment (dholbach) [inline qownnotes/snapcraft.yaml]: Can you change this to `[desktop-qt5]`?

Comment (pbek) [inline qownnotes/snapcraft.yaml]: Interesting, that worked! I will try to build with that locally and on Launchpad too! Thanks a lot, @dholbach!

**Comment by dholbach** [comment]
Does everything work now?

**Comment by pbek** [comment]
@dholbach, the local build and the build on Launchpad succeeded. Thanks a lot!

**Comment by dholbach** [comment]
Nice work!

**Comment by pbek** [comment]
Thanks, <https://github.com/ubuntu/snappy-playpen/pull/230> is needed so it can be published to the Ubuntu store.

**Pull request merged**